satpy/readers/abi_l1b.py

```diff
@@ -35,9 +35,17 @@
 class NC_ABI_L1B(NC_ABI_BASE):
     """File reader for individual ABI L1B NetCDF4 files."""
 
+    def __init__(self, filename, filename_info, filetype_info, clip_negative_radiances=None):
+        """Open the NetCDF file with xarray and prepare the Dataset for reading."""
+        super(NC_ABI_L1B, self).__init__(filename, filename_info, filetype_info)
+        if clip_negative_radiances is None:
+            clip_negative_radiances = satpy.config.get("clip_negative_radiances")
+        self.clip_negative_radiances = clip_negative_radiances
```

Comment:
My only small feeling against the way this is done is that the config value is read at Scene/Reader creation time and not at `load` time when it is actually used...but I'm not sure I care that much.

Reply:
No, none of that. I'm just saying this attribute isn't actually used until we read the data. So the user needs to make sure this config value is set at Scene init, even though the value is used at load time. I'm purely talking about the satpy.config.get part. Don't worry about it.